azure/v20.0.0/release.yaml

```
@@ -0,0 +1,80 @@
+apiVersion: release.giantswarm.io/v1alpha1
+kind: Release
+metadata:
+  annotations:
+    "helm.sh/resource-policy": keep # TODO: Remove when this release is merged to master.
+  creationTimestamp: null
+  name: v20.0.0
+spec:
+  apps:
+  - componentVersion: 3.18.0
+    name: calico
+    version: 0.2.0
+  - name: cert-exporter
+    version: 1.6.0
+  - componentVersion: 1.1.0
+    name: cert-manager
+    version: 2.4.1
+  - name: chart-operator
+    version: 2.19.0
+  - componentVersion: 0.7.6
+    name: external-dns
+    version: 2.1.1
+  - componentVersion: 1.9.7
+    name: kube-state-metrics
+    version: 1.3.0
+  - name: metrics-server
+    version: 1.2.1
+  - name: net-exporter
+    version: 1.9.2
+  - componentVersion: 1.0.1
+    name: node-exporter
+    version: 1.7.1
+  - name: cluster-autoscaler
+    version: 1.19.1
+  - name: azure-scheduled-events
+    version: 0.2.2
+  components:
+  - name: app-operator
+    releaseOperatorDeploy: true
+    version: 4.2.0
```

```suggestion
    releaseOperatorDeploy: false
    version: 5.2.0
```

@MarcelMue Can we use the latest app-operator here?

There is a fix for the chart CR watcher https://github.com/giantswarm/app-operator/pull/790

Also releaseOperatorDeploy should be false because cluster-apps-operator will manage the app CR for the per cluster instance.

---

Sure - sounds reasonable to me. Any celestial can hit the `commit` button when they do their next round of testing :)